### ISSUE TITLE
Feature request: Only warn about missing executable once

### ISSUE THREAD

**Issue by AckslD** [mfussenegger/nvim-lint#194]
If an executable is missing you'll get the warning:
```
... Error running <executable>: ENOENT: no such file or directory
```
which is useful to know that the executable is missing. But for example if you have set linting to happen on eg `InsertLeave` you'll get a lot of these warnings and in some cases I might be ssh-ing into some machine and just want to edit something quick but don't care so much if the linter is missing.

So I'm wondering if it could be possible to only warns once per session for a given linter? I think this warning actually does not come from `nvim-lint`, is that true? But could it still be possible?

**Comment by mfussenegger** [comment]
I suspect the warning comes from here:

https://github.com/mfussenegger/nvim-lint/blob/8cc31931859dc3cc187fd68509f8649599f72cba/lua/lint.lua#L111-L114

I'd accept a PR that changes it to use `vim.notify_once` if it is available

**Comment by AckslD** [comment]
@mfussenegger cool and thanks for the pointer, I opened #198 for this
